ansible/roles/sonicv2/tasks/sonic-cavm.yml

```diff
@@ -0,0 +1,48 @@
+# SONiC
+
+# Setup Platform
+#- include: platform-cavm.yml
+
+# Install Cavium Host Interface Driver
+- name: Install driver package
+  become: true
+  apt: pkg=xp80PcieEndpoint
+       state=present
+       force=yes
+
+# Setup port SKU
+- name: Setup port configuration file
+  become: yes
+  template: src=etc/ssw/{{ sonic_hwsku }}/profile.ini.j2
```

Comment:
We try to move ssw directory into specific containers in future. We don't want to add more complexity if we have workaround.

In this case, at least BRCM, MLNX don't need the new mounting point.